src/timepicker/docs/readme.md

```diff
@@ -0,0 +1,11 @@
+In the time entry field allow user to enter the time in several formats. When user have entered the value, change formatting to the 24:00 clock. 
+### The format needed to be supported are:  ###
+ * 8a = 08:00 (If last character is a, the time is am) 
+ * 8p = 20:00 (If last character is p the time is pm)
+ * 815a / p = 08:15 / 20:15
+ * 8.15a = 08:15 / 20:15
+ * 8 = 08:00
+ * 815 = 08:15
```

Comment:
we should probably also document expected behaviour if starting with 1 or 2 e.g. 125 could be either 12.50 or 01.25?